app/views/govuk_publishing_components/components/docs/breadcrumbs.yml

```diff
@@ -78,3 +78,13 @@ examples:
         url: '/education'
       - title: 'Education of disadvantaged children appended with some extra long content to make this a very very very very long taxon'
         url: '/education'
+  stop_collapsing_on_mobile:
+    data:
```

Comment:
I know it's already been explained at the top of the page, but worth adding a quick description here as well? `On a mobile screen, only the first and last breadcrumbs are shown. Show all breadcrumbs on mobile using this option.`?